editor/header/mode-switcher/index.js

```diff
@@ -12,50 +12,35 @@ import Dashicon from '../../components/dashicon';
 class ModeSwitcher extends wp.element.Component {
 	constructor() {
 		super( ...arguments );
-		this.toggle = this.toggle.bind( this );
 		this.state = {
-			opened: false
+			value: this.props.mode
```

Comment:
We don't want to duplicate the source of truth from state. We'll simply want the `render` logic to use the prop directly. I'll plan to refactor this afternoon.